backend/main/models.py

```diff
@@ -2,5 +2,34 @@
 from django.db import models
 
 
+class MRGroups(models.TextChoices):
+    PRIVACY_OFFICER = "Privacy officer"
+    FETC_MEMBER = "FETC member"
+
+
 class User(AbstractUser):
-    pass
+    @property
+    def is_privacy_officer(self):
+        return MRGroups.PRIVACY_OFFICER in [g.name for g in self.groups.all()]
+
+    @property
+    def is_fetc_member(self):
+        return MRGroups.FETC_MEMBER in [g.name for g in self.groups.all()]
+
+    def has_access_to(self, object, mrpermission):
+        """
+        Utility function to check if a user has access to a specific object,
+        with a specific permission.
+        """
+        return object in object.__class__.objects.accessible_objects(self, mrpermission)
```

Comment:
This could be (complemented by) `can_view(self, object,)` and `can_edit(self, object,)`

Personally I'd like to omit the opaque mrpermission argument wherever possible.